sdk/core/azure-core/test/ut/telemetry_policy_test.cpp

```diff
@@ -84,6 +84,8 @@ TEST(TelemetryPolicy, telemetryString)
     EXPECT_EQ(actualValue[actualValue.size() - 1], ClosingBrace);
 
     EXPECT_EQ(actualValue.substr(0, test.expectedPrefix.size()), test.expectedPrefix);
+
+    EXPECT_EQ(actualValue.substr(actualValue.length() - 10, test.expectedPrefix.size()), "Cpp201402)");
```

Comment:
It might be nice to parameterize the language version so this doesn't fail when we update the C++ language version.